tests/test_core/test_events.py

```diff
@@ -221,3 +221,97 @@ def outer_click_is_not_triggered(event):
     await inner.click()
 
     await poll(lambda: clicked.current).until_is(True)
+
```

Comment:
It would be good to add a test for a non-`^on[A-Z]` event handler too

Reply:
I do, but I put it in `test_module.py` since I could only come up with a working example using a web module. It's called `test_callable_prop_with_javacript` - see below.